src/store/modules/picklist/actions.ts

```diff
@@ -6,6 +6,12 @@ import { PicklistService } from '@/services/PicklistService'
 import { showToast } from '@/utils'
 import { hasError } from '@/adapter'
 import { translate } from '@/i18n'
+import { useUserStore } from '@hotwax/dxp-components'
+ 
+const getCurrentFacilityId = () => {
+  const currentFacility: any = useUserStore().getCurrentFacility;
```

Comment:
Directly use `useUserStore().getCurrentFacility` wherever required as we only have a couple of places where this data is used.